code/game/gamemodes/nuclear/nuclear.dm

```diff
@@ -141,7 +141,8 @@
 
 /datum/game_mode/nuclear/proc/scale_telecrystals()
 	var/danger
-	danger = GLOB.player_list.len
+	var/list/living_crew = get_living_players()
+	danger = length(living_crew)
```

Comment:
```suggestion
	var/danger = length(living_crew)
```
there's no point to initializing the var with no value and then assign it immediately after

Reply:
(dont forget to delete the initial declaration)